R/tags.R

```diff
@@ -299,14 +299,14 @@ tagAppendChild <- function(tag, child) {
 #' @rdname tag
 #' @export
 tagAppendChildren <- function(tag, ..., list = NULL) {
-  tag$children <- c(tag$children, c(list(...), list))
+  tag$children <- unname(c(tag$children, c(dots_list(...), list)))
```

Comment:
It would probably be good to have tests for the unnaming behavior, so that it doesn't accidentally creep back into the code in the future. Same for `tagSetChildren`.